packages/app/config/webpack.prod.js

```diff
@@ -40,33 +40,7 @@ module.exports = merge(commonConfig, {
     minimizer: [
       new TerserJSPlugin({
         terserOptions: {
-          parse: {
-            // We want terser to parse ecma 8 code. However, we don't want it
```

Comment:
Have the issues these comments describe been resolved in the meantime?

Reply:
As far as I understand this issue is that terser might replace old syntax with new syntax as it's faster/smaller, but breaks ie11 support, which this PR is removing anyway